frontend/src/views/References.vue

```diff
@@ -96,8 +144,12 @@ function handleSave(xml: string) {
           <ModSelectionPanel
             v-if="amendingNormXml"
             v-model="selectedModEId"
-            class="overflow-hidden"
             :norm-xml="amendingNormXml"
+            :class="[
+              'overflow-hidden',
```

Comment:
Todo: Can you also change the pointer to be crossed out? Otherwise people might just think the application is stuck